### PR TITLE
Minor edits to standardize format and language

### PR DESCRIPTION
**Pull request by shannonbux**
I took out all the "simple, easy, basic" words and reformatted a few things. @amberleyromo Would you mind reviewing this for me? A couple UX projects are coming to mind based on these edits (i.e. special visual cues for the "challenge" things, and perhaps hover states for definitions and tips.

### CODE REVIEW

File: docs/tutorial/part-one/index.md [hunk 1]
```diff
@@ -22,7 +24,7 @@ computer's terminal. Gatsby is built using Node.js.
 
 Open a terminal window. See
 [terminal instructions for Mac users](http://www.macworld.co.uk/feature/mac-software/how-use-terminal-on-mac-3608274/) and
-[terminal instructions for Windows users](https://www.quora.com/How-do-I-open-terminal-in-windows). In your terminal window, type `node --version` and hit ENTER, then `npm --version` and hit ENTER.
+[terminal instructions for Windows users](https://www.quora.com/How-do-I-open-terminal-in-windows). In your terminal window, type `node --version` and hit ENTER, then `npm --version` and hit ENTER (tip: every time you run a command, you must type a command into your terminal and then press ENTER to run the command).
```
Comment (KyleAMathews) [inline docs/tutorial/part-one/index.md]: How about:

>tip: to run a specified command, you must type the command into your terminal and then press ENTER. Then the command will run.

File: docs/tutorial/part-one/index.md [hunk 2]
```diff
@@ -36,12 +38,11 @@ recommended version for your operating system.
...
 
 Gatsby uses "starters" for starting new projects. Starters
-are partially built Gatsby sites pre-configured to help you get moving faster.
+are partially built Gatsby sites that are pre-configured to help you get moving faster.
 There are several official starters and many others contributed from the Gatsby
 community! [See the Starters page for the full list](/docs/gatsby-starters/).
 
-To install a starter, first install Gatsby's terminal program by typing the
-following command into the terminal window and hit ENTER.
+To install a starter, first install Gatsby's terminal program by running the following command:
```
Comment (KyleAMathews) [inline docs/tutorial/part-one/index.md]: Should be `CLI` not terminal. Or actually "first install Gatsby's command line program"

**Comment by shannonbux** [comment]
Oh, and @amberleyromo I just saw your note on possibly splitting this tutorial part one into two parts. I'm open to that idea--considering "Part Zero" of the tutorials (the "programming basics" part I added in a recent PR #4153 ), would you still restructure part one, and if so, how?

**Comment by amberleyromo** [comment]
@shannonbux the way you've started splitting it with the "part zero" is exactly the direction I was envisioning. I still think, though, that the concepts should be introduced as part of the walkthrough with the ultimate goal of having the dev environment pieces in place, in preparation for moving onto actually scaffolding out a site and exploring. I'll put up a separate PR for part zero edits -- I see it being quite a bit more structured (in terms of a step-by-step path). Then, like we talked about, people can use it as an introduction to read more "vetted/quality materials" if they want to know more.

**Comment by shannonbux** [comment]
@amberleyromo that sounds great! Love that idea. Should I merge this then, @KyleAMathews so that Amberley can then edit this "part zero"? Or is there a way for you to edit this PR directly, Amberley? I'm still new to github.

**Comment by amberleyromo** [comment]
🙌yep, thanks!

**Pull request merged**